fhir-persistence-jdbc/src/test/java/com/ibm/fhir/persistence/jdbc/test/util/QuantityParmBehaviorUtilTest.java

```diff
@@ -35,12 +35,12 @@
     //---------------------------------------------------------------------------------------------------------
     // Supporting Methods:
     @BeforeClass
-    public static void before() throws FHIRException {
+    public void before() throws FHIRException {
         FHIRRequestContext.get().setTenantId("quantity");
     }
 
     @AfterClass
-    public static void after() throws FHIRException {
+    public void after() throws FHIRException {
```

Comment:
Feels like we should a base abstract class that we extend

Reply:
rather orthogonal to the already-large PR here...maybe a separate tech debt issue?

the only reason i was in these classes is that one of them was causing some test failures since it wasn't resetting the request context.